gdal/ogr/ogrsf_frmts/flatgeobuf/packedrtree.cpp

```diff
@@ -0,0 +1,341 @@
+#include "packedrtree.h"
```

Comment:
non-generated file, right ? license ?

Reply:
Right, LICENSE file added to containing dir. (ICS license)

Reply:
Parts are based on https://github.com/rawrunprotected/hilbert_curves which is in public domain but I will make sure to include a notice in the relevant sections of the code anyway for this.